CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# CONTRIBUTING.md
+
+## How to start?
+
+Great, so you want to join the development!
+
+First, [set up a development environment](INSTALL.md). As a future contributor
```

Comment:
"As a future contributor"...

Reply:
Do you mind explaining? I can't see the typo (except "Use" on the next line should be lowercase, fixed).

Reply:
The sentence looks kinda cut off in the midddle

Reply:
After rendering should look like this:

![ss](https://user-images.githubusercontent.com/7026881/78511796-eb011700-778e-11ea-8ec6-6406596bd535.png)

is that OK? :thinking:

Reply:
I don't fully understand the sentence, is it equivalent to "If you're planning to develop the project on your own, the `docker-compose.dev.yml` method is recommended"?

Reply:
If you don't understand it, it's probably badly written.

Yeah, what I mean is "you are a future contributor, so use `docker-compose` method [out of many possible]".

Do you have a suggestion how to make this sentence clearer?

Reply:
What about "Since you're going to be contributing new code the `docker-compose.dev` method is recommended"?

Reply:
:thinking: too simple, we need job security.

Honestly tho, good idea. After a short fight with push --force, I've changed this to:

```
Since you're going to write new code, use the `docker-compose.dev.yml` method.
```

looks OKish?

Reply:
👍 Has this became the most over-nitpicked line in the whole codebase?

Reply:
10 commends per one line of documentation :thinking:, now this is what I call code review